packages/react-next/src/components/Link/Link.tsx

```diff
@@ -1,40 +1,10 @@
-import * as React from 'react';
-import { getGlobalClassNames, ITheme } from '../../Styling';
-import { memoizeFunction, styled } from '../../Utilities';
-import { LinkBase } from './Link.base';
-import { ILinkProps, ILinkStyleProps, ILinkStyles } from './Link.types';
+import { compose } from '@fluentui/react-compose';
+import { LinkBase } from './LinkBase';
 import * as classes from './Link.scss';
-import { css } from 'office-ui-fabric-react/lib/Utilities';
+import { LinkProps } from './Link.types';
 
-const GlobalClassNames = {
-  root: 'ms-Link',
-};
-
-const getStaticStylesMemoized = memoizeFunction(
-  (theme: ITheme, className?: string, isButton?: boolean, isDisabled?: boolean) => {
-    const globalClassNames = getGlobalClassNames(GlobalClassNames, theme);
-
-    const propControlledClasses = [isButton && classes.button, isDisabled && classes.disabled];
-
-    const rootStaticClasses = [isDisabled && 'is-disabled'];
-
-    return {
-      root: css(className, classes.root, globalClassNames.root, ...rootStaticClasses, ...propControlledClasses),
-    };
-  },
-);
-
-const getStaticStyles = (props: ILinkStyleProps): Required<ILinkStyles> => {
-  const { className, isButton, isDisabled, theme } = props;
-
-  return getStaticStylesMemoized(theme!, className, isButton, isDisabled);
-};
-
-export const Link: React.FunctionComponent<ILinkProps> = styled<ILinkProps, ILinkStyleProps, ILinkStyles>(
```

Comment:
hmm aren't you breaking a lot of logic which was in `styled` but not in `compose`? such as `Customizer` and styles pro.

Reply:
Yes, maybe this should have been started as a draft PR. I've added the `Do Not Merge` tag for now to account for this.

Reply:
i think we need to take a smaller step. use `compose` without removing `styled`

Reply:
Reverted to using `styled` for the last layer.